bundle/Form/InformationCollectionUpdateType.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\InformationCollectionBundle\Form;
+
+use eZ\Publish\API\Repository\Values\ContentType\ContentType;
+use Ibexa\Contracts\Core\SiteAccess\ConfigResolverInterface;
+use Netgen\Bundle\IbexaFormsBundle\Form\DataWrapper;
+use Netgen\Bundle\IbexaFormsBundle\Form\FieldTypeHandlerRegistry;
+use Netgen\Bundle\IbexaFormsBundle\Form\Payload\InformationCollectionStruct;
+use Netgen\Bundle\IbexaFormsBundle\Form\Type\AbstractContentType;
+use Netgen\InformationCollection\API\Value\Collection;
+use RuntimeException;
+use Symfony\Component\Form\DataMapperInterface;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+use function array_keys;
+use function in_array;
+
+class InformationCollectionUpdateType extends AbstractContentType
+{
+    protected FieldTypeHandlerRegistry $fieldTypeHandlerRegistry;
+
+    protected DataMapperInterface $dataMapper;
+
+    protected ConfigResolverInterface $configResolver;
+
+    public function __construct(
+        FieldTypeHandlerRegistry $fieldTypeHandlerRegistry,
+        DataMapperInterface $dataMapper,
+        ConfigResolverInterface $configResolver
+    ) {
+        parent::__construct($fieldTypeHandlerRegistry, $dataMapper);
+        $this->configResolver = $configResolver;
+    }
+
+    public function getName(): string
+    {
+        return $this->getBlockPrefix();
+    }
+
+    /**
+     * Returns the prefix of the template block name for this type.
+     */
+    public function getBlockPrefix(): string
+    {
+        return 'ezforms_information_collection_update';
+    }
+
+    public function configureOptions(OptionsResolver $resolver): void
+    {
+        $resolver->setRequired('collection');
+        $resolver->setAllowedTypes('collection', Collection::class);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        /** @var DataWrapper $dataWrapper */
+        $dataWrapper = $options['data'];
+        $collection = $options['collection'];
```

Comment:
This is unused, please run the CS fixer on all changed files.

Reply:
Fixed in 6fc74a5d commit